src/002_installation.md

```diff
@@ -213,3 +213,45 @@ running even if the network drops your connection as you can later
 reconnect to it. Otherwise, any network failure may require you to
 restart Tamarin and start over on the proof.
 
+Tamarin Code Editors
+--------------------
+
+Under the [etc](https://github.com/tamarin-prover/tamarin-prover/tree/develop/etc) folder contained in the Tamarin Prover project, plug-ins are available for VIM, Sublime Text 3 and Notepad++. Below we details the steps required to install your preferred plug-in.
+
+### VIM ###
+
+1. Create `.vim` directory in your home directory if not already existing
+2. Change directory to `.vim`
+3. Place the [filetype.vim]() file
+4. Create another directory named `syntaxes` in the `.vim` directory and change to it
```

Comment:
To the best of my knowledge, `$VIMRUNTIME/syntaxes`, even if it works, is not a documented feature. The VIM help states `$VIMRUNTIME/syntax` (so typically `~/.vim/syntax/` ). Also the VIM wiki that you point to uses `/syntax/` and not `/syntaxes/`. So I suggest changing it to `syntax`.